source/validate_id.cpp

```diff
@@ -972,8 +975,8 @@ bool idUsage::isValid<SpvOpSpecConstantComposite>(const spv_instruction_t* inst,
         auto constituent = module_.FindDef(inst->words[constituentIndex]);
         auto constituentOpCode = constituent->opcode();
         if (!constituent || !(SpvOpSpecConstantComposite == constituentOpCode ||
-            SpvOpConstantComposite == constituentOpCode ||
-            SpvOpUndef == constituentOpCode)) {
+                              SpvOpConstantComposite == constituentOpCode ||
```

Comment:
There's a lot of reformatting going on.  Is one tool inconsistent with another?

Reply:
Formatting is fixed, but I can't get clang-format to do the right thing.